### PR TITLE
Add translation packages, data, and translate about us page

### PR DESCRIPTION
**Pull request by roseja**
* Add translations for About Us page.
* Add language switcher to navigation menu. (Looking for some help on fixing the design for this/best practice) Edit: I think I fixed this

about-us-page-en-updated
about-us-page-ua-updated

about-us-mobile-ua

### CODE REVIEW

File: components/Nav/nav.js [hunk 1]
```diff
@@ -73,6 +86,26 @@ export default function NavBar() {
                             </a>
                         </Link>
                     ))}
+                    <ul
+                        className="w-11/12 sm:w-1/4 flex list-none mt-3 mb-4 flex-row min-w-fit rounded-full box-border border-2 border-white bg-gray-200"
```
Comment (roseja) [inline components/Nav/nav.js:90]: Does anyone have a suggestion on how to update the classes so it aligns properly?

File: components/Nav/nav.js [hunk 2]
```diff
@@ -43,11 +54,13 @@ export default function NavBar() {
                     href='/'
                     passHref
                   >
+                    <a>
```
Comment (roseja) [inline components/Nav/nav.js:57]: I added this to fix a console error I saw.

File: components/Nav/nav.js [hunk 3]
```diff
@@ -98,6 +131,30 @@ export default function NavBar() {
               ))}
             </div>
           </Disclosure.Panel>
+          <div className="sm:hidden max-w-full mx-auto">
```
Comment (roseja) [inline components/Nav/nav.js:134]: This is the mobile language switcher view

**Comment by geositta** [comment]
@roseja This looks good, thank you. Please if you would check the route/pathname using Next router to only render this language toggle on the About us page? Because users' first response will be that the language toggle is broken on pages that are not translated. You could make an array in the navbar component to map over the known translated pages and only render the toggle on the pages that have translations.

This is not an array example but you could take this and adapt it to check an array with the includes method:
const router = useRouter()
const aboutUs = router.pathname === '/about-us' && <AboutUs />

return (
     {aboutUs}
)

Please note that I get a build error when I try to build the app locally which is problematic when trying to debug build issues. I was able to build the app in Vercel so it appears like we could work around this for time being.

File: components/Nav/nav.js [hunk 4]
```diff
@@ -13,12 +13,24 @@ const navigation = [
...
 
 export default function NavBar() {
   const router = useRouter()
+  let en_lang, ua_lang;
+  [en_lang, ua_lang] = ["en", "ua"];
+
+  function changeLanguage(e) {
+    router.push(router.asPath, undefined, { locale: e.target.value, });
+  }
+  let showLanguageSwitcher = translatedpath.includes(router.pathname);
```
Comment (roseja) [inline components/Nav/nav.js:33]: Here is the array for currently translated pages.

**Comment by roseja** [comment]
I'm able to `npm run build` locally. Not sure if this is a version issue?
`npm --v
8.4.0`
`ukraine rose$ npm run build

> ukraine@0.1.0 build
> next build

info  - Loaded env from /Users/rose/Developer/daocentral/ukraine/.env
info  - Checking validity of types
info  - Creating an optimized production build
info  - Compiled successfully
info  - Collecting page data
info  - Generating static pages (70/70)
info  - Finalizing page optimization

Page                                          Size     First Load JS
┌ ● /                                         10.5 kB         126 kB
├   └ css/088d2c512f877611.css                92 B
├   /_app                                     0 B            90.4 kB
├ ● /[row] (ISR: 10 Seconds) (22718 ms)       444 B           116 kB
├   ├ /en/5 (2002 ms)
├   ├ /en/7 (2001 ms)
├   ├ /en/6 (1053 ms)
├   ├ /en/1 (993 ms)
├   ├ /en/4 (818 ms)
├   ├ /en/2 (697 ms)
├   ├ /en/8 (627 ms)
├   └ [+39 more paths] (avg 372 ms)
├ ○ /404                                      194 B          90.6 kB
├ ● /about-us                                 2.93 kB         115 kB
├ λ /api/hello                                0 B            90.4 kB
├ ○ /for-donors                               3.8 kB          116 kB
├ ● /for-fundraisers                          5.33 kB         117 kB
├ ● /for-fundraisers/for-large-organizations  3.88 kB         116 kB
├ ○ /for-fundraisers/for-reputation-backers   4.2 kB          116 kB
└ ● /for-fundraisers/for-small-fundraisers    4.61 kB         117 kB
+ First Load JS shared by all                 90.4 kB
  ├ chunks/framework-5f4595e5518b5600.js      42 kB
  ├ chunks/main-31e8b9b17b05ae4c.js           28.5 kB
  ├ chunks/pages/_app-7827170fcfb32486.js     19 kB
  ├ chunks/webpack-9b312e20a4e32339.js        836 B
  └ css/f7c025301d280cbc.css                  4.09 kB

λ  (Server)  server-side renders at runtime (uses getInitialProps or getServerSideProps)
○  (Static)  automatically rendered as static HTML (uses no initial props)
●  (SSG)     automatically generated as static HTML + JSON (uses getStaticProps)
   (ISR)     incremental static regeneration (uses revalidate in getStaticProps)`

I made the update for only showing for translated routes and put a comment near the array. Thanks for that suggestion!

**Review by geositta — APPROVED** [review]

**Pull request merged**